homeassistant/components/blink/translations/en.json

```diff
@@ -14,7 +14,7 @@
                 "data": {
                     "2fa": "Two-factor code"
                 },
-                "description": "Enter the PIN sent to your email",
+                "description": "Enter the PIN sent via email or SMS",
```

Comment:
You need to change this in `strings.json` too. That's the source of truth.

Reply:
Thanks for the heads up! Done.